iotedgedev/template/deployment.template.json

```diff
@@ -45,7 +45,7 @@
             "status": "running",
             "restartPolicy": "always",
             "settings": {
-              "image": "${CONTAINER_REGISTRY_SERVER}/filter-module:linux-x64-${CONTAINER_TAG}",
```

Comment:
Let's remove container tag from the default template.  We'll document how to use it.